src/style.css

```diff
@@ -1 +1,57 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  display: flex;
```

Comment:
Add `align-items: center` property to center the text vertically.
Also, position should be relative, so bottom line is absolute to the link